smallest_multiple.js

```diff
@@ -6,9 +6,25 @@
  * @return { Number }         Lowest Positive Number that is evenly divisible by all numbers
  *                            between 1 and `ceiling`
  */
-module.exports = function( ceiling ) {
+module.exports = function(ceiling) {
   // do work here
+  let i = ceiling;
```

Comment:
Generally do not use single character variable names outside of the the specific scenario of `for (let i = 0; ...`